app/Components/Draggable.js

```diff
@@ -34,19 +35,27 @@ export default function Draggable({
       initialX: x,
       initialY: y,
     };
+    selectedItem = { id: vector.id };
```

Comment:
Is this line of code required? I don't see it coming from another component or going to another component.

Reply:
Yeah, it's not required. Just deleted it!